packages/bpk-docs/src/components/neo/HeroSection/HeroSection.scss

```diff
@@ -0,0 +1,45 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+.bpkdocs-hero-section {
+  display: flex;
+  height: $bpk-spacing-xs * 70;
+  padding: $bpk-spacing-xxl;
+  flex-direction: column;
+  justify-content: flex-end;
+  background-color: $bpk-color-gray-700;
+  background-size: cover;
+
+  @include bpk-breakpoint-above-tablet {
+    height: $bpk-spacing-xs * 110;
+    padding: $bpk-spacing-xs * 10;
+    padding-bottom: $bpk-spacing-xs * 5;
+  }
+
+  &__heading {
+    color: $bpk-color-white;
+    white-space: pre-line;
```

Comment:
Using this so that the heading can include new lines, as we have on the copywriting page.